### PR TITLE
pin to x86_64 which is amd64 in docker speak

### PR DESCRIPTION
**Pull request by esc**
is based on #3

### CODE REVIEW

**Comment by sklam** [comment]
Is what i added in https://github.com/numba/numba-hatchery/pull/2/commits/23dcf7887a9244a11f3b77d79b99c332f78268e8 not sufficient for your setup?

**Comment by esc** [comment]
> Is what i added in [23dcf78](https://github.com/numba/numba-hatchery/commit/23dcf7887a9244a11f3b77d79b99c332f78268e8) not sufficient for your setup?

No, the `pull`, `build` and `run` commands all need to be specific about the platform for this to work on Apple Silicon.

**Comment by sklam** [comment]
> > Is what i added in [23dcf78](https://github.com/numba/numba-hatchery/commit/23dcf7887a9244a11f3b77d79b99c332f78268e8) not sufficient for your setup?
> 
> No, the `pull`, `build` and `run` commands all need to be specific about the platform for this to work on Apple Silicon.

hm... It worked fine for me though on my new m1

**Comment by sklam** [comment]
I'm referring to `FROM --platform=linux/amd64` specially that fixed it all for me

**Comment by esc** [comment]
> I'm referring to `FROM --platform=linux/amd64` specially that fixed it all for me

ah, let me try that

**Comment by esc** [comment]
> > I'm referring to `FROM --platform=linux/amd64` specially that fixed it all for me
> 
> ah, let me try that

Ah yes, it seems sufficient, only the `update` command may need the flag. Do you agree?

**Comment by sklam** [comment]
> > > I'm referring to `FROM --platform=linux/amd64` specially that fixed it all for me
> > 
> > 
> > ah, let me try that
> 
> Ah yes, it seems sufficient, only the `update` command may need the flag. Do you agree?

Yes, that's true

**Comment by esc** [comment]
@sklam I've updated this PR as requested. `--platform` now only used on `update`.

File: makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .DEFAULT_GOAL=compile
 
 update:
-	docker pull python:3.11.1-bullseye
+	docker pull --platform linux/amd64 python:3.11.1-bullseye
 build:
-	docker build --no-cache -t hatchery-bootstrap hatchery-bootstrap
+	docker build --no-cache -t  hatchery-bootstrap hatchery-bootstrap
```
Comment (sklam) [inline makefile]:
```suggestion
	docker build --no-cache -t hatchery-bootstrap hatchery-bootstrap
```

Comment (esc) [reply makefile]: fixed in 1161dd6c6f1df7e9b90c76554006ec5bb4b01704

**Review by sklam — APPROVED** [review]
LGTM

**Pull request merged**